README.md

```diff
@@ -1,3 +1,97 @@
-# Router
+# @kitbag/router
 
-Type safe router for Vue.js
+A simple and versatile mapping utility for Typescript.
+
+[![Npm Version](https://img.shields.io/npm/v/@kitbag/router.svg)](https://www.npmjs.org/package/kitbag/router)
+[![Zip Size](https://img.badgesize.io/https:/unpkg.com/@kitbag/router/dist/kitbag-router?compression=gzip)](https:/unpkg.com/@kitbag/router/dist/kitbag-router)
+[![Netlify Status](https://api.netlify.com/api/v1/badges/d6033c76-88c3-4963-b24f-7a0bda20f271/deploy-status)](https://app.netlify.com/sites/kitbag-router/deploys)
+
+Get started with the [documentation](https://kitbag-router.netlify.app/)
+
+## Installation
+
+Install Kitbag Router with your favorite package manager
+
+```bash
+# bun
+bun add @kitbag/router
+# yarn
+yarn add @kitbag/router
+# npm
+npm install @kitbag/router
+```
+
+## Define Basic Routes
+
+Create an array of possible routes. Learn more about [defining routes](/core-concepts/defining-routes).
```

Comment:
Does this link need to be fully qualified?

Reply:
yes. great catch